erpnext/controllers/accounts_controller.py

```diff
@@ -883,7 +883,10 @@ def get_default_taxes_and_charges(master_doctype, tax_template=None, company=Non
 	if tax_template and company:
 		tax_template_company = frappe.db.get_value(master_doctype, tax_template, "company")
 		if tax_template_company == company:
-			return
+			return {
+				'taxes_and_charges': tax_template,
+				'taxes': get_taxes_and_charges(master_doctype, tax_template)
+			}
```

Comment:
the first time `get_default_taxes_and_charges` is called in `make_quotation` in opportunity.py. There due to no template it goes on line 891 directly and gets default template and taxes too.
The second time around it need not fetch it again, hence return works fine.
Perhaps check why it wasnt set or if it wasnt fetched due to another confuguration. Works fine for me